test/setup/main.tf

```diff
@@ -16,12 +16,12 @@
 
 
 provider "google" {
-  version = "~> 2.12.0"
+  version = "~> 3.0"
 }
 
 module "project_ci_kms" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 3.0"
+  version = "~> 8.0"
```

Comment:
Since we are moving to `8.x` I think it makes sense to set `skip_gcloud_download=true` to speed up CI as we already have gcloud binaries in the image.